crypto/src/asset.rs

```diff
@@ -44,6 +63,19 @@ impl From<&str> for Denom {
     }
 }
 
+impl From<Denom> for ValidatorAssetId {
+    fn from(denom: Denom) -> ValidatorAssetId {
+        // Convert an asset name to an asset ID by hashing to a scalar
+        ValidatorAssetId(Fq::from_le_bytes_mod_order(
+            // XXX choice of hash function?
+            blake2b_simd::Params::default()
+                .personal(b"Penumbra_AssetID")
```

Comment:
should this be a different string? i copied this from the regular asset ID impl

Reply:
I think this should probably be replaced by a call to the normal asset ID impl: https://github.com/penumbra-zone/penumbra/pull/242/files#r765482232